test/jasmine/tests/parcoords_test.js

```diff
@@ -0,0 +1,821 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+var d3 = require('d3');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+var mock0 = { // mock with zero dimensions; special case, as no dimension can be rendered
```

Comment:
It was originally there, but if I put it in `test/image/mocks/` then an image generation *must* take place and it currently fails locally (and probably on CI as well) and of course CI complains if there's no baseline image. One of the remaining listed task items is to look into why it fails to generate a baseline image in the `docker` instance.